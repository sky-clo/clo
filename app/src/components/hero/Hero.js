import React from "react";

import styles from "./hero.module.scss";

export default function Hero(props) {
  return (
    <section
      className={styles.section}
      style={{
        backgroundImage: `linear-gradient(
        180deg,
        rgba(0, 0, 0, 0.4) 17.71%,
        rgba(0, 0, 0, 0) 100%
      ),
      url("${props.image}")`,
      }}
    >
      <h1 className="c-heading-alpha">{props.title}</h1>
      {props.description && <p className="c-text-lead">{props.description}</p>}
    </section>
  );
}
