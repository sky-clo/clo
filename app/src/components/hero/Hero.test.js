import React from "react";
import { shallow } from "enzyme";

import Hero from "./Hero";

describe("Hero", () => {
  it("should match snapshot", () => {
    const render = shallow(
      <Hero
        image="https://static.skyassets.com/contentstack/assets/bltdc2476c7b6b194dd/blt556529d1b8bf298d/5edfe30f98e62f38ec84dadd/2020_Sky_Logo.png"
        title="Sky"
        description="Europeâ€™s leading direct-to-consumer media and entertainment company"
      />
    );
    expect(render).toMatchSnapshot();
  });
});
