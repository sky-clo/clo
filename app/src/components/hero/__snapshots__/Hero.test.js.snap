// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero should match snapshot 1`] = `
<section
  className="section"
  style={
    Object {
      "backgroundImage": "linear-gradient(
        180deg,
        rgba(0, 0, 0, 0.4) 17.71%,
        rgba(0, 0, 0, 0) 100%
      ),
      url(\\"undefined\\")",
    }
  }
>
  <h1
    className="c-heading-alpha"
  />
</section>
`;
